spring-batch-core/src/test/java/org/springframework/batch/core/repository/dao/JdbcStepExecutionDaoTests.java

```diff
@@ -75,4 +76,16 @@ public void testTruncateExitDescription() {
 				.length() < stepExecution.getExitStatus().getExitDescription().length());
 	}
 
+	@Transactional
+	@Test
+	public void testCountStepExecutions() {
+		// arrange

```

Comment:
We use the "given/when/then" naming in tests (as you might have seen). There is nothing wrong with the 3 "A"s naming, but for consistency, could you please update the tests in this PR to use the current naming?

Reply:
Changed naming type for consistency